packages/cli/src/commands/deploy/index.ts

```diff
@@ -310,18 +310,11 @@ export default async (client: Client): Promise<number> => {
   const contextName = org.slug;
   client.config.currentTeam = org.type === 'team' ? org.id : undefined;
 
-  // if we have `sourceFilesOutsideRootDirectory` set to `true`, we use the current path
-  // and upload the entire directory.
-  const sourcePath =
-    rootDirectory && !sourceFilesOutsideRootDirectory
```

Comment:
Based on the previous code (before #12598), `sourceFilesOutsideRootDirectory` was essentially _always_ set to `true`, and thus `cwd` was being used as the `sourcePath` always.

So the fix in this PR is to pass `cwd` where `sourcePath` was being used before.